src/styles/_helpers.scss

```diff
@@ -1,3 +1,33 @@
-.\+flex { display: flex; }
-.\+cursor-clickable { cursor: pointer; }
-.\+hover-grey:hover{ color: $color-active-text; }
+.\+flex {
+  display: flex;
+}
+.\+cursor-clickable {
+  cursor: pointer;
+}
+.\+hover-grey:hover {
+  color: $color-active-text;
+}
+
+@function spacing($name, $true-val:false) {
+  @if $true-val == true {
+    @return map-get($spacings,$name);
+  } @else {
+    @return var(--spacing-#{$name});
+  }
```

Comment:
is this a sass thing: `var(--spacing-#{$name})` how does it work?

Reply:
whoops, these are no longer used, I'll remove them. 

I've added a lot of scss maps and parsers, to out put css custom properties (native css variables) from our scss.